src/types.jl

```diff
@@ -256,6 +256,7 @@ For example, given a gain of `20dB`, we have `val===20`. This type differs from
 struct Gain{L, S, T<:Real} <: LogScaled{L}
     val::T
 end
+Unitful.Gain{L, S, T}(x::Unitful.Gain{L, S, T}) where {L,S,T<:Real} = x # for method ambiguity on 1.9
```

Comment:
```suggestion
Unitful.Gain{L, S, T}(x::Unitful.Gain{L, S, T}) where {L,S,T<:Real} = x # for ambiguity resolution
```
The ambiguity exists on all 1.x versions, not only 1.9.